### ISSUE TITLE
[bug] debug image loading logs line formatted poorly

### ISSUE THREAD

**Issue by jrasell** [shipyard-run/shipyard#63]
When using the `nomad_cluser.image` configuration resource, debug log lines are weirdly formatted with blank lines and trailing characters. It would be nice to clean this up to improve the UX.

```
2020-05-26T16:12:38.564+0200 [DEBUG] oaded image: consul:1.7.2
2020-05-26T16:12:39.446+0200 [DEBUG]
2020-05-26T16:12:39.839+0200 [DEBUG] 7Loaded image: gcr.io/google_containers/pause-amd64:3.0
2020-05-26T16:12:40.457+0200 [DEBUG]
2020-05-26T16:12:41.161+0200 [DEBUG] 2Loaded image: nicholasjackson/fake-service:v0.9.0
2020-05-26T16:12:41.469+0200 [DEBUG]
2020-05-26T16:12:46.624+0200 [DEBUG] :Loaded image: nicholasjackson/consul-envoy:v1.7.2-v0.14.1
2020-05-26T16:12:47.550+0200 [DEBUG]
2020-05-26T16:12:57.494+0200 [DEBUG] %Loaded image: prom/prometheus:latest
2020-05-26T16:12:57.608+0200 [DEBUG]
2020-05-26T16:12:58.437+0200 [DEBUG] *Loaded image: prom/statsd-exporter:latest
```

**Comment by nicholasjackson** [comment]
I agree here @jrasell, this is coming when we stream the logs from a Docker Container using the Docker SDK. I might dig into the Docker CLI to see how they do it as I must be missing something.

**Comment by nicholasjackson** [comment]
Hey @jrasell this has been fixed in the latest build, v0.0.30 will be available as soon as the automated deploy finishes.
